helix-view/src/editor.rs

```diff
@@ -529,15 +529,17 @@ impl Default for WhitespaceCharacters {
 }
 
 #[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
-#[serde(default)]
+#[serde(default, rename_all = "kebab-case")]
 pub struct IndentGuidesConfig {
     pub render: bool,
     pub character: char,
+    pub skip: u16,
```

Comment:
Since it takes the number of levels to skip now, something like `skip_levels` might be more suited, or maybe `render_level`.

Reply:
renamed to skip_levels